src/features/claim/components/CreateWallet.tsx

```diff
@@ -54,17 +54,17 @@ export const CreateWallet = ({
     }
   };
 
-  const walletOptions = WALLET_OPTIONS
+  const walletOptions = supportedWallets
 
     // TODO replace with filter this is temporary
     // .filter((wallet) => wallets.includes(wallet.id))
-    .filter((wallet) => wallet.id === 'mynearwallet')
+    .filter((wallet) => wallet.name === 'mynearwallet')
```

Comment:
why are we filtering for MNW here? we already have the default wallet variable in the config

Reply:
this filter was added by Matt temporarily to only show My Near Wallet in claim pages